typescript-json-schema.ts

```diff
@@ -1195,14 +1195,14 @@ export function exec(filePattern: string, fullTypeName: string, args = getDefaul
 
     const definition = generateSchema(program, fullTypeName, args, onlyIncludeFiles);
     if (definition === null) {
-        return;
+        throw new Error("No output definition. Probably caused by errors prior to this?");
```

Comment:
Can you give an example that triggers this line? I want to make sure that we want to raise an error here.

Reply:
Added a test case that does so